modules/subscriptions.php

```diff
@@ -211,6 +211,11 @@ public function should_email_post_to_subscribers( $post ) {
 			return false;
 		}
 
+		// Private posts are not sent to subscribers.
+		if ( 'private' === $post->post_status ) {
```

Comment:
I'm not sure where we decide what posts are sent out to subscribers. Not `post_status=private` ones? Only `post_status=publish` ones?

Reply:
I believe that is done on WordPress.com after the sync, but @enejb would know better.